.gitignore

```diff
@@ -25,6 +25,7 @@ wheels/
 *.egg
 MANIFEST
 node_modules/
+frontend/static/frontend/main.js
```

Comment:
There are a couple of things like this where they're committed both here and on #18.. can you tidy that up?

Reply:
Yep, definitely. Looks like I didn't properly separate my branches. It was a little bit of "trying to keep the concerns separate, whilst finding a few changes as I went" with regards to this one. I'll go back over and try separate them out / put the right things in the right places.